lib/cacheable/controller.rb

```diff
@@ -37,10 +37,16 @@ def response_cache(key_data=nil, version_data=nil, &block)
       end
 
       handler = Cacheable::ResponseCacheHandler.new(self) do |h|
```

Comment:
Did you experiment yet with moving all configuration to constructor args and using the block as the miss callback?

Reply:
I found this list of args really long and ugly and kinda liked the block constructor interface so I left it as is for now.

Reply:
I changed it to your suggested format 👍